### PR TITLE
Fix wrong count condition in SQL

### PR DESCRIPTION
**Pull request by masayukig**
This commit fixes the wrong count condition in sql.py. The condition
always count it as success because

  if test_dict['status'] == 'success' or 'xfail':

is always True.

Fixes: #41

### CODE REVIEW

File: stestr/repository/sql.py [hunk 1]
```diff
@@ -244,9 +244,9 @@ def _handle_test(self, test_dict):
             else:
                 self.totals[test_dict['status']] += 1
             values = {}
-            if test_dict['status'] == 'success' or 'xfail':
+            if test_dict['status'] in ('success', 'xfail'):
                 values['passes'] = self.totals['success']
-            elif test_dict['status'] == 'fail' or 'uxsuccess':
+            elif test_dict['status'] in ('fail' or 'uxsuccess'):
```
Comment (mtreinish) [inline stestr/repository/sql.py]: Shouldn't this also be a comment?

Comment (mtreinish) [inline stestr/repository/sql.py]: err sorry not a comment, a comma

Comment (masayukig) [inline stestr/repository/sql.py]: ah, yeah, it should be.. (It seems I'm still in jet lag..)

**Comment by mtreinish** [comment]
I kinda wish we had unit tests for the sql backend. But I won't force you to add any

**Comment by masayukig** [comment]
Thanks for merging! heh, yeah, exactly. I was lazy, but I'll do it in another patch.

**Pull request merged**